github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -131,16 +131,17 @@ presubmits:
           name: vfio
       nodeSelector:
         hardwareSupport: gpu
-      podAntiAffinity:
-        preferredDuringSchedulingIgnoredDuringExecution:
-        - weight: 100
-          podAffinityTerm:
-            labelSelector:
-              matchExpressions:
-              - key: vgpu
-                operator: In
-                values:
-                - "true"
+      affinity:
```

Comment:
This did not fail validation?

Reply:
Seems like the check ignores unknown elements, since it expected the `affinity` and not `podAntiAffinity`.